evolver/device.py

```diff
@@ -70,25 +73,43 @@ def schema(self):
             "controllers": [{"kind": str(type(a)), "config": a.Config.model_json_schema()} for a in self.controllers],
         }
 
+    def _loop_exception_wrapper(self, callable, message="unknown") -> bool:
+        try:
+            callable()
```

Comment:
I wont do this for now if it is ok. It'll make meesage require to be keyword in the call, and since it is marked private I think scope is limited enough